google/cloud/security/scanner/scanners/gce_networks_scanner.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Scanner for the Networks Enforcer acls rules engine."""
+from google.cloud.security.common.util import log_util
+from google.cloud.security.common.data_access import gce_networks_dao
+from google.cloud.security.common.data_access import project_dao
+from google.cloud.security.common.gcp_type.resource import ResourceType
+from google.cloud.security.scanner.audit import gce_networking_rules_engine
+from google.cloud.security.scanner.scanners import base_scanner
+
+LOGGER = log_util.get_logger(__name__)
+
+
+class GceFirewallNetworksScanner(base_scanner.BaseScanner):
```

Comment:
From what I understand, this should be better named as "InstanceNetworkInterfacesScanner".